README.md

```diff
@@ -82,13 +82,62 @@ This will allow you to run your favourite IDE but not have to install any depend
 
 Step by step quickstart guide can be found in the full docs here https://linkfree.eddiehub.io/docs/quickstart
 
----
 
-Looking for inspiration? You can view the following profiles for an example:
+<!-- Testimonials STARTs Here -->
+
+
+## ⛩️ Testimonials
+
+
```

Comment:
Double blank lines are ignored in markdown and only a single blank line is displayed, please use standard markdown, look at the rest of the file to be consistent